rainier-compute/src/main/scala/com/stripe/rainier/compute/Evaluator.scala

```diff
@@ -2,6 +2,10 @@ package com.stripe.rainier.compute
 
 class Evaluator(var cache: Map[Real, Double]) extends Numeric[Real] {
 
+  def parseString(str: String): Option[Real] =
```

Comment:
The better way to solve this is probably to drop the `extends Numeric[Real]`, and replace references to `Numeric[Real]` elsewhere with a direct reference to `Evaluator`.

Reply:
note: decided to follow up on this in another PR.